components/bl-leaflet-map/src/helpers/map.js

```diff
@@ -86,21 +86,61 @@ export function initMap(component, eventHandlers, map, currentLayer, uid) {
   });
 }
 
+function validateCircle(circle) {
+  if (!circle.point) {
+    console.error('Circle error!\n Circle should store "point" property with latlng object in\n', circle);
+
+    return false;
+  }
+
+  const { point: { lat, lng }, radius, description } = circle;
+  const isAllPropertiesAreNumbers = [lat, lng, radius].every(value => {
+    const result = !isNaN(value);
+
+    if (!result) {
+      console.error(`Circle error!\n Expected number but received "${ value }" in\n`, circle);
+    }
+
+    return result;
+  });
+
+  if (!isAllPropertiesAreNumbers) {
+    return false;
+  }
```

Comment:
forEach?

Reply:
The foreEach method returns only `undefined`.

Reply:
We doesn't need return. forEach - isn't what we need in any case. Use `for of` to iterate and break validating at this step.